app/views/profiles/show.html.erb

```diff
@@ -4,14 +4,18 @@
   <strong>Email:</strong>
   <%= @user.email %>
 </h3>
-
 <h3>
   <strong>Role:</strong>
   <%= @user.role %>
 </h3>
+<h3>
+  <strong>Salary:</strong>
+  <%= @user.salaries.last.amount.to_s %>
```

Comment:
having a separate method to get the last salary would be more convenient. We have a lot of places in the code where it can be used. Also, just imagine that you need to change the `last` to the `first`. It would be not a simple task.